frontend/src/css/App.css

```diff
@@ -55,7 +71,12 @@ Formatting of eMIB Sample Test
 
 /*Section Titles Formatting*/
 .emib-section-titles {
-  color: #137991;
+  color: #00565e;
+}
+
+/*Subsection Titles Formatting*/
+.emib-subsection-titles {
```

Comment:
Instead of adding an extra class name, we should just override the colour of our `h3` tag.

Reply:
I added an h3 style as suggested and removed this emib-subsection-titles class in App.css.
See commit #https://github.com/code-for-canada/project-thundercat/pull/87/commits/10d4d7af20de41c34b98a6f720e3364edf745ef8